models/linkages/R/read.restart.LINKAGES.R

```diff
@@ -4,26 +4,26 @@
 ##' 
 ##' @param outdir      output directory
 ##' @param runid       run ID
-##' @param time        year that is being read
-##' @param settings    PEcAn settings object
-##' @param variables   variables to be extracted
+##' @param stop.time   year that is being read
+##' @param multi.settings    PEcAn settings object
+##' @param var.names   var.names to be extracted
 ##' 
 ##' @description Read Restart for LINKAGES
 ##' 
 ##' @return X.vec      vector of forecasts
 ##' @export
 ##' 
-read.restart.LINKAGES <- function(outdir,runid,time,settings,variables,sample_parameters=NULL){
+read.restart.LINKAGES <- function(outdir,runid,stop.time,multi.settings,var.names,params=NULL){
```

Comment:
I was going to fix SIPNET in one pull request and LINKAGES in another. Or should I just fix them both now?